examples/ssx-test-dapp/src/App.js

```diff
@@ -127,7 +127,7 @@ function App() {
         console.error(err);
       }
     } else {
-      ssxProvider.signOut();
+      ssxProvider && ssxProvider.signOut();
```

Comment:
Under what circumstances would `ssxProvider` be falsy? Want to make sure this isn't masking a potential present or future bug

Reply:
To answer your question, I don't see exactly where ssxProvider could be null, but the linter can't evaluate this. I can remove this lint change and we can evaluate this logic and make changes later, if you'd prefer @44203